tests/e2e/improved_csi_idempotency.go

```diff
@@ -228,6 +284,116 @@ var _ = ginkgo.Describe("[csi-block-vanilla] [csi-file-vanilla] "+
+	var pvclaims []*v1.PersistentVolumeClaim
+	var err error
+	pvclaims = make([]*v1.PersistentVolumeClaim, volumeOpsScale)
+
+	// Decide which test setup is available to run
+	if vanillaCluster {
+		ginkgo.By("CNS_TEST: Running for vanilla k8s setup")
+		// TODO: Create Thick Storage Policy from Pre-setup to support 6.7 Setups
+		scParameters[scParamStoragePolicyName] = "Management Storage Policy - Regular"
+		// Check if it is file volumes setups
+		if rwxAccessMode {
+			scParameters[scParamFsType] = nfs4FSType
+		}
+		curtime := time.Now().Unix()
+		randomValue := rand.Int()
+		val := strconv.FormatInt(int64(randomValue), 10)
+		val = string(val[1:3])
+		curtimestring := strconv.FormatInt(curtime, 10)
+		scName := "idempotency" + curtimestring + val
+		storageclass, err = createStorageClass(client, scParameters, nil, "", "", false, scName)
+	} else if supervisorCluster {
+		ginkgo.By("CNS_TEST: Running for WCP setup")
+		thickProvPolicy := os.Getenv(envStoragePolicyNameWithThickProvision)
+		if thickProvPolicy == "" {
+			ginkgo.Skip(envStoragePolicyNameWithThickProvision + " env variable not set")
+		}
+		profileID := e2eVSphere.GetSpbmPolicyID(thickProvPolicy)
+		scParameters[scParamStoragePolicyID] = profileID
+		// create resource quota
+		createResourceQuota(client, namespace, rqLimit, thickProvPolicy)
+		storageclass, err = createStorageClass(client, scParameters, nil, "", "", false, thickProvPolicy)
+	} else {
+		ginkgo.By("CNS_TEST: Running for GC setup")
+		thickProvPolicy := os.Getenv(envStoragePolicyNameWithThickProvision)
+		if thickProvPolicy == "" {
+			ginkgo.Skip(envStoragePolicyNameWithThickProvision + " env variable not set")
+		}
+		createResourceQuota(client, namespace, rqLimit, thickProvPolicy)
+		scParameters[svStorageClassName] = thickProvPolicy
+		storageclass, err = createStorageClass(client, scParameters, nil, "", "", false, thickProvPolicy)
+	}
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+	defer func() {
+		if !supervisorCluster {
+			err := client.StorageV1().StorageClasses().Delete(ctx, storageclass.Name,
+				*metav1.NewDeleteOptions(0))
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		}
+	}()
+
+	// TODO: Stop printing csi logs on the console
+	collectPodLogs(ctx, client, csiSystemNamespace)
+
+	// This assumes the tkg-controller-manager's auto sync is disabled
+	ginkgo.By(fmt.Sprintf("Reducing Provisioner time interval to %s Sec for the test...", customProvisionerTimeout))
+	updateCSIDeploymentProvisionerTimeout(client, csiSystemNamespace, customProvisionerTimeout)
+
+	defer func() {
+		ginkgo.By(fmt.Sprintf("Resetting provisioner time interval to %s sec", defaultProvisionerTimeInSec))
+		updateCSIDeploymentProvisionerTimeout(client, csiSystemNamespace, defaultProvisionerTimeInSec)
+	}()
+
+	ginkgo.By("Creating PVCs using the Storage Class")
+	framework.Logf("VOLUME_OPS_SCALE is set to %v", volumeOpsScale)
+	for i := 0; i < volumeOpsScale; i++ {
+		fmt.Printf("Creating %v pvc ", i)
```

Comment:
don't use `fmt.Printf`, use `framework.Logf()`
Also, consider changing the log message to "Creating pvc%v..."

Reply:
Fixed